socorro/unittest/external/postgresql/test_crashstorage.py

```diff
@@ -230,7 +227,7 @@ def test_basic_postgres_save_processed_success(self):
         crashstorage.save_processed(a_processed_crash)
 
         assert mocked_database_connection_source.call_count == 1
-        assert mocked_cursor.execute.call_count == 3
+        assert mocked_cursor.execute.call_count == 2
```

Comment:
We're not longer saving to the `processed_crashes_*` tables, so we can remove one call.